src/features/student/views/StudentToolsTracesView/components/TraceAiUsageToggleFormField/TraceAiUsageToggleFormField.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import type { CreateTraceForm, UpdateTraceForm } from '@/features/student/types'
+import { TraceAiUsageToggle } from '@/features/student/components'
+import { useAttrs } from 'vue'
+
+interface TraceAiUsageToggleFormFieldProps {
+  form: CreateTraceForm | UpdateTraceForm
+}
+
+defineProps<TraceAiUsageToggleFormFieldProps>()
+
+const emit = defineEmits<{
+  (event: 'change', payload: boolean): void
+}>()
+
+const attrs = useAttrs()
+
+function handleChange (value: boolean, fieldChange: (value: boolean) => void) {
+  fieldChange(value)
+  emit('change', value)
+}
+</script>
+
+<template>
+  <form.Field name="useIA">
+    <template #default="{ field }">
+      <TraceAiUsageToggle
+        v-bind="attrs"
+        id="use-ia"
+        name="useIA"
+        :model-value="field.state.value"
+        @update:model-value="(value) => handleChange(value, field.handleChange)"
+      />
+      <div
+        v-if="field.state.meta.errors.length > 0"
+        class="declaration-items__error"
+      >
+        {{ field.state.meta.errors.join(', ') }}
```

Comment:
```suggestion
        <span class="caption-regular something-error">{{ field.state.meta.errors.join(', ') }}</span>
```

Et du coup ajouter le style pour la couleur sur something-error

Reply:
@p-beger c'est outdated ça, j'ai viré declaration-items__error carrement prsk ce toggle n'est pas required 
c'est un truc zapé